dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/group/ComponentGroupApi.kt

```diff
@@ -40,9 +45,10 @@ interface ComponentGroupApi {
  */
 inline fun <reified T : ComponentBase> ComponentGroupApi.create(
     identifier: String,
+    insertBeforeIdentifier: String? = null,
     noinline init: (T.() -> Unit)? = null,
 ): T {
-    return this.create(identifier, T::class, init)
+    return this.create(identifier, insertBeforeIdentifier, T::class, init)
```

Comment:
What is this insertBeforeIdentifier?

Reply:
The `insertBeforeIdentifier` property can be used to specify the order at which an element is to be inserted into the section (i.e., add a custom component just before the component with identifier X)

Reply:
Thats rather cool!